apps/smbmount.sh

```diff
@@ -123,23 +131,35 @@ do
             # If not remove this line from fstab
             msg_box "It seems like the mount wasn't successful. It will get deleted now. Please try again.\nAs a hint:\n- you might fix the connection problem by enabling SMB3 on your SMB-server.\n- You could also try to use the IP-address of the SMB-server instead of the Server-name, if not already done.\n- Please also make sure, that 'ping IP-address' of your SMB-Server from your Nextcloud-instance works."
             sed -i "/$SMBSHARES_SED\/$count /d" /etc/fstab
+            if [ -f $SMB_CREDENTIALS/SMB$count ]
+            then
+                check_command rm $SMB_CREDENTIALS/SMB$count
+            fi
             break
         else
-            # Install and enable files_external
-            if ! is_app_enabled files_external
+            # Check if Nextcloud is existing
+            if [ -f $NCPATH/occ ]
```

Comment:
Maybe add this to all the functions tha use 'occ' instead?

Reply:
What do you mean by that?

Reply:
add:

```
if [ -f $NCPATH/occ ]
then
    some_occ_command
fi
```

...in each function that use `occ`

Reply:
So shall I add this to the occ_command or occ_command_no_check functions?
Is that your suggestion?

concerning the script: the only places where I use them is already covered by this one check.

Reply:
@enoch85 I need an answer here before merging..